Themis/Views/Assessment/Pane/RightGripView.swift

```diff
@@ -33,6 +33,7 @@ struct RightGripView: View {
             if paneVM.dragWidthRight > 0 {
                 Color.themisPrimary
                     .frame(maxWidth: 7, maxHeight: .infinity)
+                    .ignoresSafeArea()
```

Comment:
Did the safe area cause issues with the grip? I wouldn't have thought of this!

Reply:
Yes, the vertical line was cut at the bottom of the screen due to the safe area 🙂